os_net_config/sriov_config.py

```diff
@@ -595,12 +592,13 @@ def add_udev_rule_for_vdpa_representors(pf_name):
         vadd = VF_PCI_RE.search(att.get('device'))
         if not vadd:
             logger.error(
-                f"{att.get('device')}/{vf}: Failed to get pf/vf numbers "
-                "and so failed to create a udev rule for renaming vdpa dev"
+                f"{pf_name}: {att.get('device')}/{vf}: Failed to get "
+                "pf/vf numbers and so failed to create a udev rule for "
+                "renaming vdpa dev"
             )
             continue
         vdpa_rep = f"vdpa{vadd.group(1)}p{vadd.group(2)}vf{vadd.group(3)}"
-        logger.info(f"{vdpa_rep}: Adding udev representor rule.")
+        logger.info(f"{pf_name}: {vdpa_rep} Adding udev representor rule.")
         udev_lines += (
             'SUBSYSTEM=="net", ACTION=="add", '
             f'ATTR{{address}}=="{mac}", NAME="{vdpa_rep}"\n'
```

Comment:
f-string missing at line 610 -> logger.info("adding udev rules to unmanage vf representators")

Reply:
Not required